components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodCLIService.kt

```diff
@@ -65,7 +69,11 @@ class GitpodCLIService : RestService() {
             val file = parseFilePath(fileStr) ?: return "invalid file"
             val shouldWait = getBooleanParameter("wait", urlDecoder)
             return withClient(request, context) {
-                CommandLineProcessor.doOpenFileOrProject(file, shouldWait).future.get()
+                GlobalScope.launch {
+                    withContext(Dispatchers.IO) {
+                        CommandLineProcessor.doOpenFileOrProject(file, shouldWait).future.get()
+                    }
+                }
```

Comment:
ℹ️ This change is required by the latest IDE build because `doOpenFileOrProject` is now a `suspend function`.